versions.tf

```diff
@@ -2,7 +2,7 @@ terraform {
   required_version = ">= 0.12.0, < 0.14.0"
 
   required_providers {
-    aws   = "~> 2.0"
+    aws   = "~> 3.0"
```

Comment:
aws    = ">= 3.0, < 4.0"

Let's add the ceiling as well to prevent breaking the module when the provider is updated to the major v4 (we saw a lot of issues when the aws provider got updated to a major version)